public_html/wp-content/themes/pattern-directory/src/components/pattern-thumbnail/canvas.js

```diff
@@ -11,6 +11,16 @@ import Iframe from '../iframe';
 const VIEWPORT_WIDTH = 800;
 const ASPECT_RATIO = 2 / 3;
 
+/**
+ * Returns the height of the preview window.
+ *
+ * @param {HTMLElement} element Html element
+ */
+
+export const getFrameHeight = ( element ) => {
+	return element.clientWidth * ASPECT_RATIO;
+};
+
```

Comment:
What do you think about moving this & `ASPECT_RATIO` to `src/utils`?

Reply:
That could work... the calculation will still need to match though, making the assumption that `clientWidth` would always be used presents potential debt.

Reply:
Yeah, I meant moving it to the utils folder and importing that function here & in card-skeleton -- in fact, if you made `getFrameHeight` take a width instead of an HTMLElement, you can use it [here too](https://github.com/WordPress/pattern-directory/blob/ff09c1afe4fad91686b9d41b508b5fc4cdfa7c13/public_html/wp-content/themes/pattern-directory/src/components/pattern-thumbnail/canvas.js#L41), and then `ASPECT_RATIO` could stay "internal" to that utils folder.

Reply:
Oh right, yeah totally, will do.